### PR TITLE
#158423628 Add passengers' landing page

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?

* Add passengers' landing page. A passenger should able to see all currently open rides by all drives. They should have access to a feature that enables them to request any available ride (link) 

#### Description of Task to be completed?

* Building (without functionality), the UI elements with HTML&CSS. 

#### How the app should be tested
On a webbrowser:

* [Download](https://github.com/artorious/Ride-My-Way/archive/ft-passengers-landing-page-158423628.zip) the repo
* Extract and navigate into the `Ride-My-Way` folder
* in the `UI` folder, open `passenger_landing.html` on a browser

On a  terminal (Linux):

* Install and set up git `sudo apt install git` 

* Clone the repository `git clone https://github.com/artorious/Ride-My-Way.git`

* Run `git checkout ft-passengers-landing-page-158423628`

* Run `cd Ride-My-Way/UI/`

* Open `passenger_landing.html` on a browser

#### Any background context you want to provide?
The UI has no functionality. 
Developed using HTML and CSS

#### What are the relevant pivotal tracker stories?
[#158423628](https://www.pivotaltracker.com/story/show/158423628)

#### Screenshots 

passland

#### Questions:

### CODE REVIEW

File: UI/static/main.css [hunk 1]
```diff
@@ -120,4 +150,9 @@ footer a {
     .container {
         width: 75%;        
     }
+}
+
+/* Handle wide tables */
+.long-x-axis {
+    overflow-x: auto;
 }
```
Comment (LWanjiru) [inline UI/static/main.css]: Add new line

File: UI/passenger_landing.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <td>Nairobi CBD</td>
+        <td>Mai-Mahiu</td>
+      </tr>
+    </table>
+  </div>
+  <hr>
+  <footer>&copy;RideMyWay 2018.<br><a href="#">Contact Us.</a></footer>
+</div>
+</body>
+</html>
```
Comment (LWanjiru) [inline UI/passenger_landing.html]: add a new empty line here

**Review by LWanjiru — APPROVED** [review]
@artorious I'm a little curious as to how this would look/work in a mobile app. 

Otherwise, looks good, can be merged

File: UI/static/main.css (hunk 1 above)
Comment (artorious) [reply UI/static/main.css]: I will add `\n`. 
Regarding the wide tables on small devices, I will try to implement collapsing tables next time. I'm counting on the grey `highlight` on `hover` will aid the user to keep track of selected row as they scroll horizontally.

**Pull request merged**